src/model/cloud-runner/aws/index.ts

```diff
@@ -0,0 +1,101 @@
+import * as SDK from 'aws-sdk';
+import CloudRunnerSecret from '../services/cloud-runner-secret';
+import CloudRunnerEnvironmentVariable from '../services/cloud-runner-environment-variable';
+import CloudRunnerTaskDef from '../services/cloud-runner-task-def';
+import AWSTaskRunner from './aws-task-runner';
+import { CloudRunnerProviderInterface } from '../services/cloud-runner-provider-interface';
+import BuildParameters from '../../build-parameters';
+import CloudRunnerLogger from '../services/cloud-runner-logger';
+import { AWSJobStack } from './aws-job-stack';
+import { AWSBaseStack } from './aws-base-stack';
+import { Input } from '../..';
+
+class AWSBuildEnvironment implements CloudRunnerProviderInterface {
+  private baseStackName: string;
+
+  constructor(buildParameters: BuildParameters) {
+    this.baseStackName = buildParameters.awsBaseStackName;
+  }
+  async cleanupSharedResources(
+    // eslint-disable-next-line no-unused-vars
+    buildGuid: string,
+    // eslint-disable-next-line no-unused-vars
+    buildParameters: BuildParameters,
+    // eslint-disable-next-line no-unused-vars
+    branchName: string,
+    // eslint-disable-next-line no-unused-vars
+    defaultSecretsArray: { ParameterKey: string; EnvironmentVariable: string; ParameterValue: string }[],
+  ) {}
+  async setupSharedResources(
+    // eslint-disable-next-line no-unused-vars
+    buildGuid: string,
+    // eslint-disable-next-line no-unused-vars
+    buildParameters: BuildParameters,
+    // eslint-disable-next-line no-unused-vars
+    branchName: string,
+    // eslint-disable-next-line no-unused-vars
+    defaultSecretsArray: { ParameterKey: string; EnvironmentVariable: string; ParameterValue: string }[],
+  ) {}
```

Comment:
I assume these are intended for future use right?

(just making sure it's not a leftover)

Reply:
Used in Kuberentes BuildProvider and will probably be used in AWS at some point to speed things up.

It is a required part of the BuildProviderInterface